src/map/handler/Map.ScrollWheelZoom.js

```diff
@@ -20,14 +21,13 @@ L.Map.ScrollWheelZoom = L.Handler.extend({
 	removeHooks: function () {
 		L.DomEvent.off(this._map._container, {
 			mousewheel: this._onWheelScroll,
-			MozMousePixelScroll: L.DomEvent.preventDefault
+			MozMousePixelScroll: this._preventScroll
```

Comment:
No, that's why I wrote `if (!delta) { delta = L.DomEvent.getWheelDelta(e); }` a bit further down.

On second thought, handling the (deprecated) `MozMousePixelScroll` event fit better in `L.DomEvent` instead.